main/options.c

```diff
@@ -57,7 +57,7 @@
 #define EXTENSION_SEPARATOR '.'
 #define PATTERN_START '('
 #define PATTERN_STOP  ')'
-#define IGNORE_SEPARATORS   ", \t\n"
```

Comment:
> Because macro replacements with parameters generally contain spaces.

How about ','? 

macor replacements with parameters can contain spaces, can't they?